ignite/templates/app/stargate/go.mod.plush

```diff
@@ -3,26 +3,199 @@ module <%= ModulePath %>
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.5
-	github.com/cosmos/ibc-go/v3 v3.0.1
+	github.com/cosmos/cosmos-sdk v0.46.0
+	github.com/cosmos/ibc-go/v5 v5.0.0-beta1
 	github.com/gogo/protobuf v1.3.3
 	github.com/golang/protobuf v1.5.2
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
-	github.com/ignite/cli v0.23.0
-	github.com/spf13/cast v1.4.1
-	github.com/spf13/cobra v1.4.0
-	github.com/stretchr/testify v1.7.1
-	github.com/tendermint/spn v0.2.1-0.20220708132853-26a17f03c072
-	github.com/tendermint/tendermint v0.34.19
+	github.com/ignite/cli v0.23.1-0.20220801134230-8d30121393da
```

Comment:
I had dependency issues when testing the [blogclient tutorial](https://docs.ignite.com/guide/blog/connect-blockchain#creating-a-blockchain-client). The `go mod tidy` didn't work until I changed the `go.mod` in the [blog chain app](https://docs.ignite.com/guide/blog) to use a different CLI version:
```
github.com/ignite/cli v0.23.0
```

The checksum that I got after that is:
```
github.com/ignite/cli v0.23.0 h1:k8jDwG+hMd54KDvSrnEbo76ANM4NT9B9ReuuK/3md20=
github.com/ignite/cli v0.23.0/go.mod h1:8HH/rwaLsPis2M7wAyAQOG8j4MhpdWmtWC4lR8MoV5s=
```

I also changed the **blog client** `go.mod` to match the same CLI version.